stackdriver_trace.c

```diff
@@ -30,7 +30,7 @@
 #include <sys/time.h>
 #endif
 
-// True global for storing the original zend_execute_ex function pointer
+/* True globals for storing the original zend_execute_ex and zend_execute_internal function pointers */
```

Comment:
I'm not seeing any consistency within the PHP source. In general, there aren't really any comments but I have found some comment lines as long as 272 characters. That doesn't mean we can't be better.